packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -36,13 +40,15 @@ import { stringToBoolean } from "../../shared/types";
 import { stringIntegerSchema, stringListFromQuerySchema } from "../schemas/shared";
 import { getUUIDFrom, uuidSchema } from "../schemas/uuid";
 import { asyncHandler, getFrom, getFromParamsOrFail, getFromQueryAsArrayOrFail } from "../util";
-import { dtoFromCW, PatientLinksDTO } from "./dtos/linkDTO";
+import { PatientLinksDTO, dtoFromCW } from "./dtos/linkDTO";
 import { getResourcesQueryParam } from "./schemas/fhir";
 import { linkCreateSchema } from "./schemas/link";
 
 dayjs.extend(duration);
 
 const router = Router();
+const patientChunkSize = 25;
+const SLEEP_TIME = dayjs.duration({ seconds: 1 });
```

Comment:
If NABD, I'd recommend a higher sleep, like 5s. Just because a patient triggers other async work and we don't want to overload the API too much. And this is done by us manually, so we can run a cxId per time or something like that, so its not to bad to wait a little longer to avoid overloading the API.

Reply:
Sounds good